### ISSUE TITLE
Selenium 

### ISSUE THREAD

**Issue by nousacademy** [kesla/node-headless#15]
Would it be possible to run this with Selenium as a headless browser?? As shown here (http://elementalselenium.com/tips/38-headless), except in nodejs??

**Comment by kesla** [comment]
Yeah, I can't think of a reason why you wouldn't be able to do that.

**Comment by kesla** [comment]
node-headless & the ruby headless gem are quite similar.

**Comment by nousacademy** [comment]
Thanks! I'll try it out, wish there was a nodejs tutorial on this way, or maybe ill have to make one haha

**Comment by arnauorriols** [comment]
Hi. I have the same need. The principal difference between node-headless and the ruby headless gem is the interface; the ruby version uses start() and stop(), whereas node-headless uses a CPS approach. With the latter, I can't achieve the same pattern as described in the tutorial linked above, where the start() is called in the before() hook, and the stop() is called in the after() hook. Does anyone have an alternative pattern for using node-headless within a test suite?

**Comment by sean-hill** [comment]
Hey @nousacademy, did you ever solve this problem? I'm looking to use [this selenium](https://github.com/vvo/selenium-standalone) repo to run on my linux server, but I know I need `xvfb` running in the background, just not sure how.

**Comment by nousacademy** [comment]
Hi @sean-hill I actually didn't get a chance to do it. I was trying it this was as an alternative to JSdom, because I could never install JSdom properly. When I do it I'll post an example.
